pkg/config/application_test.go

```diff
@@ -10,10 +10,12 @@ import (
 	"bytes"
 	"testing"
 
+	"github.com/hyperledger/fabric/common/tools/protolator"
+	"github.com/hyperledger/fabric/common/tools/protolator/protoext/peerext"
+
```

Comment:
nit: empty line in import.

Reply:
agreed; we don’t need to separate these imports into a group.